backend/cmd/headlamp.go

```diff
@@ -275,6 +275,23 @@ func createHeadlampHandler(config *HeadlampConfig) http.Handler {
 
 	config.handleClusterRequests(r)
 
+	r.HandleFunc("/customapiproxy", func(w http.ResponseWriter, r *http.Request) {
```

Comment:
Instead of custom (here and in the rest of the cases), I think it makes more sense to use `external`.
And we still need to have this turned off by default, pending the new server option we have discussed.

Reply:
I made a similar suggestion, I think "/proxy" and the query param "proxy-to" makes more sense.